<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitialMesh</class>
 <widget class="QDialog" name="InitialMesh">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Initial Mesh Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="InitialMeshTW">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>500</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Mesh Control</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="6" column="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="11" column="3" colspan="3">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Save as default</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="4">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>            Mesh Size of Model Parts (mm)：</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="3">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Model part parameters：</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="3">
        <widget class="QTableWidget" name="tableWidget_model">
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>150</number>
         </attribute>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Model parts：</string>
         </property>
        </widget>
       </item>
       <item row="10" column="4" colspan="3">
        <widget class="QLineEdit" name="lineEdit_meshPart"/>
       </item>
       <item row="5" column="2" colspan="2">
        <widget class="QLineEdit" name="lineEdit_bcd"/>
       </item>
       <item row="0" column="5" colspan="2">
        <widget class="QLineEdit" name="wavelength">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
         <property name="text">
          <string>10</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="MinimumgridsofperwavelengthL">
         <property name="text">
          <string>Boundary objects：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="4">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Boundary object parameters：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QTableWidget" name="tableWidget_boundary">
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>150</number>
         </attribute>
        </widget>
       </item>
       <item row="0" column="0" colspan="5">
        <widget class="QLabel" name="MinimumgridsoftheadjacentpointsL">
         <property name="text">
          <string>Basic mesh size (mm)：</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Coarest Distance(mm)：</string>
         </property>
        </widget>
       </item>
       <item row="4" column="6">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Finest Mesh Times：</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="2">
        <widget class="QLineEdit" name="lineEdit_bcgt"/>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Coarest Mesh Times：</string>
         </property>
        </widget>
       </item>
       <item row="4" column="7">
        <widget class="QLineEdit" name="lineEdit_bgt"/>
       </item>
       <item row="5" column="6">
        <widget class="QLabel" name="label_bfd">
         <property name="text">
          <string>Finest Distance(mm)：</string>
         </property>
        </widget>
       </item>
       <item row="5" column="7">
        <widget class="QLineEdit" name="lineEdit_10"/>
       </item>
       <item row="8" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="OKPB">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="CancelPB">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
